frontend/online-shop/src/components/productsList/DescriptionCard.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button, GridListTileBar } from "@material-ui/core";
+import { useStyle } from "../../styles/productsList/DescriptionCardStyle";
+import { ThemeProvider } from "@material-ui/styles";
+import { standardTheme } from "../../themes/StandardTheme";
+import { Link } from "react-router-dom";
+
+export interface Product {
+  id: bigint;
+  name: string;
+  price: number;
+  weight: number;
+  category: string;
+  supplier: string;
+  imgUrl: string;
+}
+
+export const DescriptionCard = (props: Product) => {
+  const style = useStyle();
+
+  return (
+    <ThemeProvider theme={standardTheme}>
```

Comment:
If you have provided the theme as a wrapper within the root component (`App.tsx`) you don't need to also provide it in the child components.